docs/journey/day1/README.md

```diff
@@ -41,6 +41,8 @@ The `kam bootstrap` [command](../../commands/kam_bootstrap.md) also provides an
 
 In the event of using a self-hosted _GitHub Enterprise_ or _GitLab Community/Enterprise Edition_ if the driver name isn't evident from the repository URL, use the `--private-repo-driver` flag to select _github_ or _gitlab_.
 
+The personal access token `--git-host-acccess-token` passed in the bootstrap command will be stored in a secure manner in the keyring on your local file system, if the secret with the same host-name  `Eg. github.com` is present in the keyring, the flag will overwrite the current value present in the keyring with the updated git-host-access-token.
+
```

Comment:
We need to add description in the bootstrap/webhook commands as well.   We need to state that for commands (bootstrap/webhook) that need token, the following locations are searched (in this sequence)
* When a token is provided in command flag, the token will be stored securely in keyring.  The hostname of the URL (e.g. github.com) will be used to stored the token by keyring.  The provided token will be used.
* When a token is not provided in command flag, the token is retrieved by keyring using the hostname of the URL.  If no token is not found, a token is retrieved from env var.   What is the format of the env var?   If no token is not found in the env var, the command will fail.